### ISSUE TITLE
shmup orientation

### ISSUE THREAD

**Issue by xmack** [arcade-mini/FBA-A320-RS07#1]
Hi you can add a rotate screen option? Al lot of shmups loads vertical and no way to change it.
Thanks for your work!

**Comment by gameblabla** [comment]
No plans for it for now. (i think porting FBA-GCW0 might be a better idea) Which game exactly btw ?

**Comment by xmack** [comment]
For example : 19xx, dimanhoo,mercs,varth

**Comment by xmack** [comment]
I tried edit a cfg file for roms, modify the rom parameters using de emulator menu but always are on horizontal position  for play it with rotated handheld but in rs07....
If you could add this option working to the menu configuration it would be great!

**Comment by gameblabla** [comment]
It won't work because it is simply not implemented in FBA-A320, no need to mess around with the menu.

**Comment by xmack** [comment]
Any idea of how to solve it'?

**Comment by gameblabla** [comment]
Yeah but this will require several lines of code and i don't feel like working on FBA for now after i had just added fullscreen scaling.

**Comment by xmack** [comment]
Ok if you can do it in future updates it will be great! Thanks for your work.

**Comment by xmack** [comment]
i don't understand nothing. I modify the cfg for example for the game 19xx.zip Like this:
# FBA config file
# 19xx

FBA_SOUND 2
FBA_SAMPLERATE 0
FBA_VSYNC 0
FBA_RESCALE 0
FBA_ROTATE 1
FBA_SHOWFPS 0
FBA_FRAMESKIP -1
FBA_M68KCORE 0
FBA_Z80CORE 0
FBA_SENSE 100

Change fba rotate to 1. Start the game and nothing changes but the modified cfg.file returns to fba_rotate 0 when i opened again
